cl/attestation/attestation.go

```diff
@@ -0,0 +1,128 @@
+package attestation
+
+import (
+	"context"
+	"sync"
+
+	"github.com/ledgerwatch/erigon-lib/kv"
+	"github.com/ledgerwatch/erigon/cl/clparams"
+	"github.com/ledgerwatch/erigon/cl/cltypes"
+	"github.com/ledgerwatch/erigon/cl/cltypes/solid"
+	state_accessors "github.com/ledgerwatch/erigon/cl/persistence/state"
+	"github.com/ledgerwatch/erigon/cl/sentinel"
+)
+
+type Attestation struct {
+	indiciesDB   kv.RoDB
+	beaconConfig *clparams.BeaconChainConfig
+	netConfig    *clparams.NetworkConfig
+	sentinel     *sentinel.Sentinel
```

Comment:
use sentinel.SentinelClient interface, https://github.com/ledgerwatch/interfaces/blob/master/p2psentinel/sentinel.proto and add the require methods there